modules/ROOT/pages/_partials/rn-known-issues-ref.adoc

```diff
@@ -8,4 +8,4 @@ This section lists known issues with  {PRODUCT} {PATCH_VERSION_NO}:
 // * You do not have the required privileges to add {walkthrough-name}s to a cluster. If you require this feature, contact Red Hat Support for a workaround.
 
 * Any {PRODUCT_SHORT} user can determine what {walkthrough-name}s are available in a cluster.
-In a future release, this ability will be restricted to the customer-admin user.
+In a future release, this ability will be restricted to the `customer-admin` user.
```

Comment:
I wonder
- should we be less specific about the restriction
- should we create some way to track this for later?

Reply:
I think future features should be left out.

Reply:
It's a known issue, we do talk about future releases there, right?
I'm happy with it in this case, because the issue is so egregious